sokoban/src/player.ts

```diff
@@ -0,0 +1,31 @@
+export function handleBoard(e: any, newGameBoard: any, player: any, x: any, y: any, target: any) {
+  let copy = [...newGameBoard];
+  let newX = x;
+  let newY = y;
+
+  switch (e.key) {
```

Comment:
arrow up should be y -1 arrow down y +1

Reply:
yeah, what happens in the mapArray check for player X, Y -> Y is the first value and X later because of the mapping

Reply:
Guys, it is another feature...